R-package/DESCRIPTION

```diff
@@ -45,11 +45,12 @@ NeedsCompilation: yes
 Biarch: true
 VignetteBuilder: knitr
 Suggests:
+    DiagrammeR,
     knitr,
     markdown,
     processx,
     RhpcBLASctl,
-    testthat
+    testthat,
```

Comment:
```suggestion
    testthat
```

@fboudry I suspect this trailing comma is why `R CMD check` is failing. I don't think trailing commas in lists are supported in `DESCRIPTION` (just as they are not supported in R).

Let's remove this.